### DISCUSSION TITLE
Task description not showing.

### DISCUSSION THREAD

**Discussion by Jaz0314** [Vdauphin/HeartsAndMinds#1636] [Question and answer] [answered]
So I’ve been working on making my own side mission. So far I’ve been able to get everything to work perfectly except the text for the description doesn’t show up. The space is just blank. The symbol for the task shows but has no text. I’ve created two new Id’s for the task(case 42 and case 43) and made new strings for them. I began the new strings on line  1733. Is that ok or do the new strings need to go somewhere specific in the stringtable.xml or is it something else I’m doing wrong?

**Comment by Vdauphin** [comment, marked as answer]
Hello,

Looks good for me
Remember to exit the editor menu until the A3 main menu, stringtable are not updated sometimes

Cheers
